book.common/src/main/java/com/happystays/book/common/dto/responsemodel/BookResponse.java

```diff
@@ -0,0 +1,29 @@
+package com.happystays.book.common.dto.responsemodel;
+
+import com.happystays.book.common.dto.commandmodel.PnrInfo;
+import com.happystays.book.common.dto.commonmodel.ResponseStatus;
+import com.happystays.cqrs.core.dto.responsemodel.BaseResponse;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class BookResponse extends BaseResponse {
+    private String id;
```

Comment:
Is it possible move this to BaseResponse or else you can remove id cz already we have id in command